import.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
+  env: {
+    es2020: true
+  },
+  parserOptions: {
+    sourceType: 'module'
+  },
+  plugins: [
+    // https://github.com/import-js/eslint-plugin-import
+    'import'
+  ],
+  rules: {
+    // if the file path for an import can't be resolved locally error
+    // this rule ignores namespaced npm packages
+    // (this seems to be broken in the plugin)
+    'import/no-unresolved': ['error', {ignore: ['^@']}],
```

Comment:
As mentioned before, this one might be a problem with our bedrock "only install peer deps in the test dir" approach.